src/doc/gui_manual/Operators/OperatorTypes/PICS/Poincare_operator.rst

```diff
@@ -0,0 +1,318 @@
+.. _Poincare operator:
+
+Poincare operator
+~~~~~~~~~~~~~~~~~
+
+The Poincare operator constructs a Poincaré section for toroidal geometry, i.e.
+for streamlines/fieldlines that lie on toroidal geometry (cylindrical
```

Comment:
check spelling of `fieldlines` ([this ref](https://computing.llnl.gov/projects/starsapphire-data-driven-modeling-analysis/poincaré-plots) uses two words)